index.html

```diff
@@ -172,43 +171,65 @@ <h2>Glossary</h2>
+
+<p><dfn data-lt="character encoding|character encoding form" class="lint-ignore export">Character encoding</dfn> or, more formally, a <em>character encoding form</em>. The way a [=coded character set=] is mapped to bytes for manipulation in a computer. Commonly referred to as just the <span class="alt_name">encoding</span>. For examples and further descriptions see <a href="https://www.w3.org/International/articles/definitions-characters/index#charsets">Character encodings: Essential concepts</a>.</p>
+
+<p><dfn class="lint-ignore export">Character set</dfn> or <span class="alt_name">repertoire</span>. The set of characters one might use for a particular purpose – be it those required to support Western European languages in computers, or those a Chinese child will learn at school in the third grade (nothing to do with computers).</p>
 
-<p><dfn id="def_case_folding" class="lint-ignore export">Case folding</dfn> is the process of making two texts which differ only in case identical for comparison purposes, that is, it is meant for the purpose of string matching. This is distinct from <a href="#def_case_mapping" class="termref">case mapping</a>, which is primarily meant for display purposes. As with the default case mappings, Unicode defines default case fold mappings ("case folding") for each Unicode code point. Unicode defines two forms of case folding.</p>
+<p><dfn class="lint-ignore export">Circumgraph</dfn>. A single vowel code point that produces glyphs on more than one side of its consonant base. </p>
 
-<p><dfn data-lt="case-sensitive|Case sensitive matching" id="def_case_sensitive" class="lint-ignore export">Case sensitive matching</dfn>. Code points are compared directly with no case folding.</p>
+<p><dfn class="lint-ignore">CJK</dfn>. An abbreviation for Chinese, Japanese, and Korean. Sometimes CJKV is used to include the Han characters used in Vietnamese.</p>
 
-<p><dfn id="def_coded_character_set" class="lint-ignore export">Coded character set</dfn>. A set of characters where a unique number has been assigned to each character. Units of a coded character set are known as <a href="#def_code_point" class="termref">code points</a>.</p>
+<p><dfn class="lint-ignore export">Coded character set</dfn>. A set of characters where a unique number has been assigned to each character. Units of a coded character set are known as [=code points=].</p>
 
-<p><dfn id="def_code_point" class="lint-ignore export">Code point</dfn>. A code point value represents the position of a character in a coded character set. For example, the code point for the letter <span class="qchar">á</span> in the Unicode coded character set is 225 in decimal, or 0xE1 in hexadecimal notation. Hexadecimal notation is commonly used for referring to code points. See also <a class="termref" href="#def_unicode_code_point">Unicode code point</a>.</p>
+<p><strong><em>CLDR</em></strong>, see [=Common Locale Data Repository=].</p>
```

Comment:
This one seems tricky. I know we already have CLDR in the glossary, but it's also a reference that one can reference directly.

Reply:
I think that a reference to CLDR will link you to Common Locale Data Repository.

Reply:
I think my comment was unclear. Should we define CLDR in the glossary, given that it is an available reference? If we do define it as a term, should we export it?